### PR TITLE
GLT-2535: added Attachments section of user manual

### PR DESCRIPTION
**Pull request by djcooke**

### CODE REVIEW

File: docs/user_manual/attachments.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+{% include userman-toplink.md %}
+
+
+
+{% assign sub = sub | plus: 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Downloading Attachments" %}
+
+The Attachments section of the Edit page for any item that allows attachments lists all of the files that are attached
+or linked to the item. To download any of these files, click its filename in the list.
```
Comment (callunity) [inline docs/user_manual/attachments.md]: The first sentence is a bit hard to parse; maybe "Files can be downloaded on the Edit page of an item. The Attachments section on the Edit page lists all of the files that are attached or linked to the item. [...]"
(Mostly, I don't think you need to specify "for any item that allows attachments" and could perhaps get away with just deleting that and leaving the rest intact)

File: docs/user_manual/attachments.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+
+
+{% assign sub = sub | plus: 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Deleting Attachments" %}
+
+You can click the trash can icon in the "Delete" column for any attachment that you'd like to delete. Only the user who
+originally attached the file, or a MISO admin can delete an attachment. Deleting the file from an item will remove the
+link to the file that is stored on the MISO server. If the file is not linked to any other items, it will be deleted
+from the MISO server as well. If a file has been attached to multiple items, then deleting it from one item will not
```
Comment (apmasell) [inline docs/user_manual/attachments.md]: Please but bold on that "not"
